### PR TITLE
add Email on yellow pages and update fields of yellow pages entities.

### PR DESCRIPTION
**Pull request by Grunze1s**
Create yellowPagesEmail Entity and related with yellow pages with manytoone relation 
made is_emergency of YellowPagesPhoneNumber default to false 
made phone_number of YellowPagesPhoneNumber bigInt from Int 
add address field in YellowPagesAddress
made subsequent changes in resolver, dto, service, module

### CODE REVIEW

File: src/schema.gql [hunk 1]
```diff
@@ -1186,10 +1206,10 @@ input UpdateYellowPagesInput {
 
 input UpdateYellowPagesPhoneNumberInput {
   """"""
-  is_emergency: Boolean
+  is_emergency: Boolean = false
 
   """Phone number"""
-  phone_number: Int
+  phone_number: Float
```
Comment (shovan777) [inline src/schema.gql:1212]: int why not???

**Review by shovan777 — COMMENTED** [review]
Please review the feedback.

File: src/schema.gql (hunk 1 above)
Comment (Grunze1s) [reply src/schema.gql:1212]: int only support 32 signed integer so to receive large we need float but bigint scalar is not provided by graphql so float is directly used by graphql schema which supports double precision floating point which is larger than int.
https://docs.nestjs.com/graphql/scalars

File: src/schema.gql [hunk 2]
```diff
@@ -225,10 +239,10 @@ input CreateYellowPagesInput {
 
 input CreateYellowPagesPhoneNumberInput {
   """"""
-  is_emergency: Boolean!
+  is_emergency: Boolean = false
 
   """Phone number"""
-  phone_number: Int!
+  phone_number: Float!
```
[the remark this answers is not in the document]
Comment (Grunze1s) [reply src/schema.gql:245]: int only support 32 signed integer so to receive large we need float but bigint scalar is not provided by graphql so float is directly used by graphql schema which supports double precision floating point which is larger than int.
https://docs.nestjs.com/graphql/scalars

File: src/schema.gql [hunk 3]
```diff
@@ -1351,7 +1400,7 @@ type YellowPagesPhoneNumber {
   is_emergency: Boolean!
 
   """Phone number"""
-  phone_number: Int!
+  phone_number: Float!
```
[the remark this answers is not in the document]
Comment (Grunze1s) [reply src/schema.gql:1403]: int only support 32 signed integer so to receive large we need float but bigint scalar is not provided by graphql so float is directly used by graphql schema which supports double precision floating point which is larger than int.
https://docs.nestjs.com/graphql/scalars

**Comment by Grunze1s** [comment]
in future we might need to create custom saclar for bigint.

File: src/schema.gql (hunk 2 above)
[the remark this answers is not in the document]
Comment (shovan777) [reply src/schema.gql:245]: Please use this [library](https://the-guild.dev/graphql/scalars/docs/scalars). It has most of the scalars we need. I think it is already included in packages.
https://the-guild.dev/graphql/scalars/docs/scalars/big-int

**Review by shovan777 — COMMENTED** [review]
Use [bigint](https://the-guild.dev/graphql/scalars/docs/scalars/big-int) for `BigInt` datatype like phonenumber

**Review by shovan777 — CHANGES_REQUESTED** [review]
Please make changes

**Review by shovan777 — APPROVED** [review]
Looks great :100:

**Pull request merged**